libs/application/templates/transport-authority/license-plate-renewal/src/fields/PlateField/PlateRadioField.tsx

```
@@ -71,10 +73,9 @@ export const PlateRadioField: FC<
                   message={
                     <Box>
                       <BulletList>
-                        {!!plate.validationErrorMessages?.length &&
-                          plate.validationErrorMessages?.map((error) => {
-                            return <Bullet>{error.defaultMessage}</Bullet>
-                          })}
+                        {plate.validationErrorMessages?.map((error) => {
+                          return <Bullet>{error.defaultMessage}</Bullet>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)